<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN-L2504" ignored="0" name="GET Feeds - Net worth Summary Insight" tests="14" failures="7" timestamp="14 Jun 2021 18:38:43 GMT" time="885.548" errors="0">
  <testcase name="testGetNotificationsForNetworthSummary" time="135.87" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="101.892" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="9.485" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="59.674" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="8.73" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="8.092" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="53.525" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="InValid createdDate either format is not correct OR created date may be greater than current date">
      <![CDATA[java.lang.AssertionError: InValid createdDate either format is not correct OR created date may be greater than current date
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:315)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="65.339" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="66.02" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="66.5" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="68.218" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="66.18" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="66.303" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="99.168" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
</testsuite> <!-- GET Feeds - Net worth Summary Insight -->
